src/store/api/ContactUsAPI.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import requestConfig from "./config";
+import { IContactUs } from "../../interfaces";
+const BASE_URL = process.env.REACT_APP_API_ENDPOINT as string;
+
+class ContactUsAPI{
+    
+    static createContactUs(contactData: IContactUs): Promise<IContactUs> {
+        return axios.get(`${BASE_URL}/contactUs`, contactData, requestConfig);
+    }
+
+    static getselectedContactUsId(_id : string): Promise <IContactUs> {
+        return axios.get(`${BASE_URL}/contactUs/${_id}`, requestConfig);
```

Comment:
It should be ${BASE_URL}/admin/contactus/.... instead of what you have use. And make sure to do the same for others except create contact us.